sklearn/metrics/base.py

```diff
@@ -20,10 +20,6 @@
 from ..utils.multiclass import type_of_target
 
 
-class UndefinedMetricWarning(UserWarning):
```

Comment:
Just noting that these were used directly from the `metrics.base` and were not imported into `metrics.__init__`!